rohmu/inotify.py

```diff
@@ -52,20 +58,20 @@ def parse_inotify_buffer(event_buffer):
 
 
 class InotifyWatcher(Thread):
-    def __init__(self, compression_queue):
+    def __init__(self, compression_queue: Queue[dict[str, str]]) -> None:
```

Comment:
`Dict`? Or maybe `Mapping` is better since it's an inbound partameter

Reply:
That would be the only use of Mapping in rohmu. For this PR I'd prefer using the built-in, if later we want to use Mapping and other more generic types we can go hunt all the places where we could change those and do so.